src/components/learning-guide-chart.coffee

```diff
@@ -65,45 +70,21 @@ module.exports = class LearningGuideChart
 
 
   drawTitle: (container, guide) ->
+    that = @
     wrap = container.append('g')
       .append('svg:text')
       .attr('text-anchor','middle')
       .attr('x', WIDTH/2)
       .attr('y', 5)
       .attr('class', 'main-title')
-      .text("Your Flight Path | #{guide.title} | All Topics")
-
-
-  drawStaticImages: (container, points) ->
-    @addImage(CLOUDS_PATH, width:10, x: 35, y: 10)
-    @addImage(CLOUDS_PATH, width:16, x: 77, y: 20)
-
-
-  drawCircles: (container, fields, points) ->
-    wrap = container.append('g')
-      .attr('class', 'circles')
-       .selectAll('g')
-       .data(fields)
-
-    circles = wrap.enter()
-      .append('g')
-      .attr('transform', (f,i) ->
-        "translate(#{points[i].x},#{points[i].y})"
-      )
-      .on('click', (field) =>
-        @navigateToPractice(field)
+      .text("Your Flight Path | #{guide.title} | All Topics | ")
+      .append('svg:a')
+      .attr('class', 'show-course noselect')
```

Comment:
Just an opinion (but open to discussion!) Since all elements already have a class on them, using a mixin ends up being more descriptive when just editing the styling; you don't need to check the JS to see if this element *sometimes* has `noselect` or *always* has `noselect` on it (maybe not the best example). Admittedly, when using a 3rd party lib (like bootstrap) there is no other option. @Fredasaurus thoughts? 

Whatever the decision, my vote goes for consistency!